VocabularyTrainer/Home Screen/ViewModel/HomeViewModel.swift

```diff
@@ -20,10 +20,8 @@ final class HomeViewModel {
         if let languages = UserDefaults.standard.array(forKey: UserDefaultKeys.languages) as? [String] {
             retVal = languages.map {
```

Comment:
For some reason, existing languages are not loaded anymore.

When setting a breakpoint here (line 21) on `master` it's stopping after the `if let` with the languages being there. But not on this branch for some reason they are not loaded anymore 🤷 

I don't really get why though when looking at it. It does not look as if you renamed the keys used for loading the user default values or something.

Do you have a guess what might be the issue?

Reply:
ah haha 😄 

probably this is causing it: 
<img width="204" alt="image" src="https://github.com/misteu/VocabularyTraining/assets/35889530/3fb5f01c-45b0-4b5b-8cce-5430a1e2a9ad">

Please change this back to the original bundle-id as well :)

Reply:
Done